src/PAModel/Checksum/ChecksumMaker.cs

```diff
@@ -21,8 +21,11 @@ namespace Microsoft.PowerPlatform.Formulas.Tools
     /// </summary>
     public class ChecksumMaker
     {
+        // If the checksum is generated with C6 version then treat it as if it's missing.
+        public static string VersionC6 = "C6";
```

Comment:
Could we make this more generic? Parse the version and if its <= the current one, treat it as if it's missing?

Reply:
do we want to loosen up that much? Earlier we were very strict, we required the exact version